src/Sql/Billing/Tables/ProviderPlan.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE [dbo].[ProviderPlan] (
+    [Id]             UNIQUEIDENTIFIER NOT NULL,
```

Comment:
To ensure that no duplicate rows exist for a combination of `ProviderId` and `PlanType` we should add a constraint like `CONSTRAINT [PK_ProviderPlanType] UNIQUE ([ProviderId],[PlanType])`

Reply:
@r-tome Great idea; I've added this in this commit: https://github.com/bitwarden/server/pull/3875/commits/6447b4e75a1a2f83119077c014604f2ac0093d08